cupyx/scipy/sparse/construct.py

```diff
@@ -543,3 +543,38 @@ def kron(A, B, format=None):
     data = data.ravel()
 
     return coo.coo_matrix((data, (row, col)), shape=out_shape).asformat(format)
+
+
+def kronsum(A, B, format=None):
+    """Kronecker sum of sparse matrices A and B.
+
+    Kronecker sum is matrix sum defined as sum of two Kronecker products
+    kron(I_n, A) + kron(B, I_m), where I_n and I_m are identity matrices
+
+    Args:
+        A (cupyx.scipy.sparse.spmatrix): a sparse matrix.
+        B (cupyx.scipy.sparse.spmatrix): a sparse matrix.
+        format (str): the format of the returned sparse matrix.
+
+    Returns:
+        cupyx.scipy.sparse.spmatrix:
+            Generated sparse matrix with the specified ``format``.
+
+    .. seealso:: :func:`scipy.sparse.kronsum`
+
+    """
+    A = coo.coo_matrix(A)
+    B = coo.coo_matrix(B)
+
+    if A.shape[0] != A.shape[1]:
+        raise ValueError('A is not square matrix')
+
+    if B.shape[0] != B.shape[1]:
+        raise ValueError('B is not square matrix')
+
+    dtype = sputils.upcast(A.dtype, B.dtype)
+
+    L = kron(eye(B.shape[0], dtype=dtype), A, format=format)
+    R = kron(B, eye(A.shape[0], dtype=dtype), format=format)
+
+    return (L + R).asformat(format)
```

Comment:
Nitpick: Aren't both L and R already in the target format? We fixed it in #4536.
```suggestion
    return L + R
```

Reply:
This change is causing Jenkins failure. After reverting this change, all test works fine.

Reply:
I wonder why...Anyway, let's revert it. Thanks!